src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter(Boolean);
+
+  const styleObject = {};
+
+  stylePairs.forEach((pair) => {
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. The `forEach` method is an exception to the rule against modifying existing arrays/objects, but since we're building a new object, consider using `reduce` to transform the array of style pairs directly into an object.